test/plugins/iamauth.js

```diff
@@ -44,7 +44,7 @@ const MOCK_ACCESS_TOKEN = 'eyJraWQiOiIyMDE3MDQwMi0wMDowMDowMCIsImFsZyI6IlJTMj' +
 'OX-qNFjdNUGnRpU7FmULy0r7RxLd8mhG-M1yxVzRBAZzvM63s0XXfMnk1oLi-BuUUTqVOdrM0KyY' +
 'MWfD0Q72PTo4Exa17V-R_73Nq8VPCwpOvZcwKRA2sPTVgTMzU34max8b5kpTzVGJ6SXSItTVOUdA' +
 'ygZBng';
-const MOCK_OIDC_TOKEN_RESPONSE = {
+const MOCK_TOKEN_RESPONSE = {
```

Comment:
Should we change it to `MOCK_IAM_TOKEN_RESPONSE` which is used in python-cloudant?

Reply:
Yeah, good spot. I have updated and squashed.